python/src/usecases/branin_function.py

```diff
@@ -56,18 +55,17 @@ def __init__(self):
         self.trueNoiseFunction = 0.1
 
         self.branin = ot.SymbolicFunction(['x1', 'x2'],
-                                          ['((x2-(5.1/(4*pi_^2))*x1^2+5*x1/pi_-6)^2+10*(1-1/(8*pi_))*cos(x1)+10-54.8104)/51.9496',
-                                           str(self.trueNoiseFunction)])
+                                          ['((x2-(5.1/(4*pi_^2))*x1^2+5*x1/pi_-6)^2+10*(1-1/(8*pi_))*cos(x1)+10-54.8104)/51.9496'])
         self.transfo = ot.SymbolicFunction(['u1', 'u2'],
                                            ['15*u1-5', '15*u2'])
 
         self.model = ot.ComposedFunction(self.branin, self.transfo)
-
+        self.noiseModel = ot.SymbolicFunction(['x1', 'x2'], [str(self.trueNoiseFunction)])
```

Comment:
Huh? The "true" noise function is actually constant with value 0.1? Then there is zero difference between `metamodelNoise` and `noiseModel` in `plot_ego.py`. My earlier suggestions assumed the 2 were different. Why did we go the the trouble of using `setNoiseModel` originally, since it reset the `noiseModel_` function with... the same function?!

Reply:
we could remove the noiseModel not to confuse users too much if you want

Reply:
That might be best.

Reply:
ok